src/Components/SearchBar/index.jsx

```diff
@@ -20,10 +20,15 @@ const SearchBar = (props) => {
           placeholder="Search the word..."
           onFocus={showSuggestions}
           onBlur={hideSuggestions}
+          onChange={(e) => {
+            props.setSearchTerm(e.target.value);
+          }}
         />
         {/* <button onClick={() => props.setSearch(true)}>Search</button> */}
```

Comment:
Let's remove commented codes. These should exist in the final code only if absolutely not avoidable. Eg: to convey some information.